spec/unit/plugins/solaris2/network_spec.rb

```diff
@@ -103,7 +106,18 @@
 destination: default
        mask: default
     gateway: 10.13.37.1
-  interface: e1000g0
+  interface: e1000g0 index 3
```

Comment:
is `interface e1000g0` still a valid line for non-zones?  and if you leave the old spec test  in place and run it against your code didn't you break this by not making "index" optional in the regexp?